code/modules/chemistry/Reagents-Drugs.dm

```diff
@@ -490,6 +490,82 @@ datum
 				..()
 				return
 
+		drug/solipsizine
+			name = "solipsizine"
+			id = "solipsizine"
+			description = "A highly potent hallucinogenic substance that causes intense delirium and acute inability to percieve others."
+			reagent_state = LIQUID
+			depletion_rate = 0.1
+			addiction_prob = 8
+			fluid_r = 200
+			fluid_g = 120
+			fluid_b = 120
+			transparency = 50
+			var/counter = 1
+			var/list/invisible_people = list()
+			var/image/invisible_img = null
```

Comment:
This could be a local variable since we're storing the images in invisible_list.